src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -112,13 +112,15 @@ export class Evaluator {
   }
 
   evaluateCells(positions: CellPosition[]) {
+    const start = performance.now();
     const cellsToCompute = this.createEmptyPositionSet();
     cellsToCompute.addMany(positions);
     const arrayFormulasPositions = this.getArrayFormulasImpactedByChangesOf(positions);
     cellsToCompute.addMany(this.getCellsDependingOn(positions));
     cellsToCompute.addMany(arrayFormulasPositions);
     cellsToCompute.addMany(this.getCellsDependingOn(arrayFormulasPositions));
     this.evaluate(cellsToCompute);
+    console.info("evaluateCells (ms):", performance.now() - start);
```

Comment:
Every other logging is plain English but here it's the implementation name. I'd use plain English everywhere
 ```suggestion
     console.info("evaluate cells (ms):", performance.now() - start);
```